### DISCUSSION TITLE
Error with Firefox when adding search provider

### DISCUSSION THREAD

**Discussion by PonchoPowers** [Dotnet-Boxed/Templates#118] [Q&A]
When adding the search provider for the website to Firefox, the following error message is displayed:

Firefox could not download the search plugin from:
http://localhost:10762/opensearch.xml

**Comment by RehanSaeed** [comment]
I don't think it works on localhost. Have you tried a site on the web?

**Comment by PonchoPowers** [comment]
Not yet, I will set something up soon to give this a try and report back my findings. Thanks.

**Comment by RehanSaeed** [comment]
Any updates?

**Comment by RehanSaeed** [comment]
Feel free to re-open this if you find something.
